src/components/404-page/404Page.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import "./404Page.css"
+import NotFoundImg from "../../assets/image/404.png"
+
+
+
+const NotFoundPage = () => {
+    return (
+        <div className="notfound-container">
+            <h1>The page you are looking for is not available.</h1>
+            <div className="notfound-img">
+                <img src={NotFoundImg} alt="mentorpaired"/>
```

Comment:
Not a big deal, but if you would be adding `alt` to images, please use a more descriptive text, since this is what screenreaders use for accesibility.

Reply:
Okay, boss. I'll add it.